Platform/CommonBoardPkg/Tools/GpioDataConvert.py

```diff
@@ -2,7 +2,7 @@
 #  This is a python script for generating required output format
 #  of Gpio pin information from a given input format
 #
-# Copyright (c) 2019, Intel Corporation. All rights reserved. <BR>
+# Copyright (c) 2019 - 2020, Intel Corporation. All rights reserved. <BR>

```

Comment:
It looks this file moves under CommonBoardPkg/Tools. This is new directory, right?
In my opinion, if this is a common tool, 'BootloaderCorePkg/Tools' would be right place instead of having another location. @mauricema Any comment about the tool/script location?

Reply:
For GPIO related tool , I guess it is OK to put under Platform folder currently.  
If we do have enough confidence for this script to work for all platforms going forward, I am also fine to promote it to BootloaderCorePkg/Tools.

Reply:
Will add it in Platform folder for now.